app/helpers/interfaces_helper.rb

```diff
@@ -4,4 +4,8 @@ def interfaces_types
       [_(nic_type.humanized_name), nic_type]
     end
   end
+
+  def provided_mac?(nic)
```

Comment:
Can we make the method name bit more descriptive?
```suggestion
  def nic_with_provided_mac?(nic)
```